Bundle/CoreBundle/Resources/doc/SETUP.md

```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
+    bundles: ~
+    filters:
+        cssrewrite: ~
+        less: ~ #Victoire needs less under v3.0.0
+
+fos_user:
+    db_driver: orm
+    firewall_name: main
+    user_class: Victoire\UserBundle\Entity\User
+    from_email:
+        address: hey@troopers.email
+        sender_name: Troopers
 
-On Mac OS X:
+snc_redis:
+    clients:
+        victoire:
+            type: predis
+            alias: victoire
+            dsn: %victoire_redis_path%
+            logging: false
+
+stof_doctrine_extensions:
+    default_locale: en_US
+    orm:
+        default:
+            tree: true
+            sluggable: true
+            timestampable: true
+            softdeleteable: true
+
+victoire_core:
+    user_class: "Victoire\\UserBundle\\Entity\\User"
+    business_entity_debug: true
+    layouts:
+        defaultLayout: "Default layout"
+
+victoire_i18n:
+    available_locales:
+        - fr
+        - en
+    locale_pattern: parameter #domain
 
-```bash
-brew install redis
+    #if locale_pattern is domain, then you'll need to define the following parameter:
+    #locale_pattern_table: %locale_pattern_table%
 ```
 
-On Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-redis-on-ubuntu-16-04
+`app/config/security.yml`
+```
+security:
+    encoders:
+        Victoire\Bundle\UserBundle\Entity\User: bcrypt
+    providers:
+        fos_userbundle:
+            id: fos_user.user_provider.username
+    firewalls:
+        dev:
+            pattern: ^/(_(profiler|wdt)|css|images|js)/
+            security: false
+        main:
+            pattern: ^/
+            form_login:
+                provider: fos_userbundle
+                csrf_provider: form.csrf_provider
+                failure_path: /login
+                check_path: /login_check
+                default_target_path: /
+            logout: true
+            anonymous: true
+            switch_user: ~
+
+    access_control:
+        - { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+```
 
-- Setup Redis:
+`app/config/services.yml`
+```
+services:
+    twig.extension.text:
+       class: Twig_Extensions_Extension_Text
+       tags:
+            - { name: twig.extension }
+
+    twig.extension.intl:
+        class: Twig_Extensions_Extension_Intl
+        tags:
+            - { name: twig.extension }
+```
 
-```yml
-snc_redis:
-   clients:
-       victoire:
-           type: predis
-           alias: victoire
-           dsn: redis://localhost
+`app/config/parameters.yml.dist`
+```
+parameters:
+    ...
+    router.request_context.host: https://troopers.agency
+    router.request_context.scheme: https
+    router.request_context.base_url: '/'
+    fos_js_base_url: https://troopers.agency
+    victoire_redis_path: redis://127.0.0.1:6379
+    #if locale_pattern is domain
+    locale_pattern_table:
+        agency.troopers.dev: fr
+        troopers.agency: fr
 ```
 
-- Start Redis server:
+Think to update the `parameters.yml` with correct values.
```

Comment:
Remove *Think to*? It implies that it has to be done in the future, but it has to be done immediately.